WinFormsUI/MainForm.cs

```diff
@@ -46,24 +48,33 @@ private void MainForm_Load(object sender, EventArgs e)
 			dateTimePickerStart.Value = new DateTime(DateTime.Now.Year - 1, DateTime.Now.Month, DateTime.Now.Day);
 			dateTimePickerEnd.Value = DateTime.Now.Date;
 
-			const string stateFileName = "state.json";
-			if (!File.Exists(stateFileName))
-			{
-				File.WriteAllText(stateFileName, StateManager.SaveToJson());
-			}
-			StateManager.LoadState(File.ReadAllText(stateFileName));
+			LoadCategories();
 			LoadTransactions();
-			
+
+			File.WriteAllText(categoriesFileName, StateManager.SaveCategoriesToJson());
+			var formattedRecords = State.Instance.Transactions.Select(DisplayManager.FormatLedgerRecord);
+			File.WriteAllLines(transactionsFileName, formattedRecords);
+
 			RefreshCategories();
 			RefreshChart();
 			RefreshList();
 		}
 
+		private void LoadCategories()
+		{
+			if (!File.Exists(categoriesFileName))
+			{
+				File.WriteAllText(categoriesFileName, StateManager.SaveCategoriesToJson());
+			}
+			StateManager.LoadCategories(File.ReadAllText(categoriesFileName));
+		}
+
 		private void LoadTransactions()
 		{
 			var currentDirecory = Directory.GetCurrentDirectory();
-			var files = Directory.GetFiles(currentDirecory + "/pb").Select(f => ("pb", (Stream)File.OpenRead(f)));
-			StateManager.LoadTransactions(files);
+			//var files1 = Directory.GetFiles(currentDirecory + "/usb").Select(f => ("usb", (Stream)File.OpenRead(f)));
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5924375)